### PR TITLE
portal-web: customizable logo and text

### PR DESCRIPTION
**Pull request by JasonChen9**

### CODE REVIEW

**Comment by ddadaal** [comment]
this pr fixes #4

File: apps/portal-web/src/pageComponents/dashboard/CustomizableLogoAndText.tsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+
+interface Props {
+  hostname: string | undefined;
+}
+export const CustomizableLogoAndText: React.FC<Props> = ({ hostname }) => {
+
+  return (
+    <div>
+      <div style={{ position: "relative", width: "100%", paddingBottom: "8%" }} >
```
Comment (ddadaal) [inline apps/portal-web/src/pageComponents/dashboard/CustomizableLogoAndText.tsx:14]: we are mostly using [styled-components](https://styled-components.com/) for styling.

File: apps/portal-web/src/pageComponents/dashboard/CustomizableLogoAndText.tsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        />
+      </div>
+      <div style={{
+        display: "flex",
+        justifyContent: "center",
+        marginTop: "3%",
+        marginLeft: "20%",
+        marginRight: "20%",
+        flexDirection: "column",
+      }}>
```
Comment (ddadaal) [inline apps/portal-web/src/pageComponents/dashboard/CustomizableLogoAndText.tsx:29]: styled-components

**Review by ddadaal — CHANGES_REQUESTED** [review]

**Comment by JasonChen9** [comment]
Using styled-components for styling

**Pull request merged**